scripts/config/cspell/dictionaries/people.txt

```diff
@@ -260,3 +261,8 @@ Kawashima
 Kento
 Ueda
 Toshinari
+Josza
+Umesh
+Deustch
```

Comment:
```suggestion
Deutsch
```

Reply:
Josza is also a legit typo. Btw, both names should already be in our dictionary so it's a duplicate to add them again